src/components/EventFilters.svelte

```diff
@@ -28,6 +31,16 @@ function onLocationInput(event) {
   filteredMeetingsByLocation(query)
 }
 
+function onMyEventsChange(event) {
+  if (event.detail) {
+    setFilters({ participant: 1 })
```

Comment:
help me understand what `participant: 1` gets us here, it's not immediately apparent to me.

Reply:
I'm laughing at that :-D (if I'm understanding you correctly, it was to save 3 characters on the url?) so I think we should see if there's a different way to communicate that for later maintainers, if not, we'll just need a note explaining it.

Reply:
`participating` here might be more expressive.